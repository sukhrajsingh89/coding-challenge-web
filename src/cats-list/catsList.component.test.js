import React from "react";
import {render,cleanup } from "@testing-library/react";
 import CatsList from "./catsList.component";


describe('<CatsList',()=>{

    afterEach(()=>{
        cleanup()
    })

const renderComponent = (props={}) =>{
    return render (<CatsList {...props}/>)
}

it('it should render the component',()=>{
    const {asFragment}=renderComponent()
    expect(asFragment()).toMatchSnapshot()
})

})