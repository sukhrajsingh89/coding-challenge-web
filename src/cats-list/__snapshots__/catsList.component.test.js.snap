// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CatsList it should render the component 1`] = `
<DocumentFragment>
  <div>
    Male
  </div>
  <div
    class="sc-bdfCDU dJjNDu"
  />
  <div
    class="sc-bdfCDU dJjNDu"
  />
  <div>
    Female
  </div>
  <div
    class="sc-bdfCDU dJjNDu"
  />
</DocumentFragment>
`;
